src/server/server_family.cc

```diff
@@ -1355,14 +1355,10 @@ void PrintPrometheusMetrics(const Metrics& m, DflyCmd* dfly_cmd, StringResponse*
     absl::StrAppend(&resp->body(), replication_lag_metrics);
   }
 
-  if (m.replica_reconnections) {
-    auto& replica_reconnections = m.replica_reconnections.value();
-    AppendMetricHeader("replica_reconnect_count", "Number of replica reconnects",
-                       MetricType::COUNTER, &resp->body());
-    AppendMetricValue("replica_reconnect_count", replica_reconnections.reconnect_count,
-                      {"replica_host", "replica_port"},
-                      {replica_reconnections.host, absl::StrCat(replica_reconnections.port)},
-                      &resp->body());
+  if (m.replica_side_info) {
+    auto& replica_info = *m.replica_side_info;
+    AppendMetricWithoutLabels("replica_reconnect_count", "Number of replica reconnects",
```

Comment:
we do not need labels on this metric, so removing replica_host, replica_port labels